content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list-client.md

```diff
@@ -0,0 +1,14 @@
+---
+title: WatchListClient
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Allow an API client to get a stream of individual items instead of
+chunking from the server.
```

Comment:
How I'd explain it:
```suggestion
Enable serving collections (responses to the **list** verb) from the watch cache within the
API server, rather than always from etcd.

You can disable this feature gate if you want to select the legacy behavior where data in the
watch cache is not used to serve collections.
```